gm4_mountaineering/data/gm4_mountaineering/functions/main.mcfunction

```diff
@@ -0,0 +1,8 @@
+# Run functions to provide relevant effects when a piece of mountaineering gear is worn
+
+execute as @a[gamemode=!spectator,gamemode=!adventure,predicate=gm4_mountaineering:wearing_skis] run function gm4_mountaineering:ski_effects/skis_equipped
+execute as @a[gamemode=!spectator,gamemode=!adventure,predicate=gm4_mountaineering:holding_poles] run function gm4_mountaineering:ski_effects/poles_equipped
+execute as @a[gamemode=!spectator,gamemode=!adventure,predicate=gm4_mountaineering:wearing_crampons] run function gm4_mountaineering:ski_effects/crampons_equipped
```

Comment:
All of these should work adventure mode.
The functions should also be executed in the correct context here, consider adding `at @s` location setters where necessary and remove the context setters in the called functions.